Content.Client/Roles/RoleCodewordComponent.cs

```diff
@@ -0,0 +1,23 @@
+namespace Content.Client.Roles;
+
+/// <summary>
+/// 
+/// </summary>
+[RegisterComponent]
+public sealed partial class RoleCodewordComponent : Component
+{
+    [DataField]
+    public Dictionary<string, CodewordsData> RoleCodewords = new();
+}
```

Comment:
this needs to be serverside and replicated or else this will just break when you disconnect from the game. Network messages shouldn't be used to replicate state.